### PR TITLE
docs: update Tabs api description

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | docs: update Tabs api description |
| 🇨🇳 Chinese | docs: update Tabs api description |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
原来的 API 越看越难受，还得到处解释什么时候生效。最好是：

`<Tabs type="card" editable={{ addIcon, removeIcon }} />`

**Comment by li-jia-nan** [comment]
> 原来的 API 越看越难受，还得到处解释什么时候生效。最好是：
> 
> `<Tabs type="card" editable={{ addIcon, removeIcon }} />`

@MadCcc 要不开个投票？

File: components/tabs/index.en-US.md [hunk 1]
```diff
@@ -48,15 +48,15 @@ Common props ref：[Common props](/docs/react/common-props)
 | Property | Description | Type | Default | Version |
 | --- | --- | --- | --- | --- |
 | activeKey | Current TabPane's key | string | - |  |
-| addIcon | Customize add icon | ReactNode | `<PlusOutlined />` | 4.4.0 |
+| addIcon | Customize add icon, Only works while `type="editable-card"` | ReactNode | `<PlusOutlined />` | 4.4.0 |
```
Comment (MadCcc) [inline components/tabs/index.en-US.md:51]:
```suggestion
| addIcon | Customize add icon, only works with `type="editable-card"` | ReactNode | `<PlusOutlined />` | 4.4.0 |
```

File: components/tabs/index.en-US.md [hunk 2]
```diff
@@ -48,15 +48,15 @@ Common props ref：[Common props](/docs/react/common-props)
...
 | animated | Whether to change tabs with animation. | boolean \| { inkBar: boolean, tabPane: boolean } | { inkBar: true, tabPane: false } |  |
 | centered | Centers tabs | boolean | false | 4.4.0 |
 | defaultActiveKey | Initial active TabPane's key, if `activeKey` is not set | string | - |  |
 | hideAdd | Hide plus icon or not. Only works while `type="editable-card"` | boolean | false |  |
 | indicator | Customize `size` and `align` of indicator | { size?: number \| (origin: number) => number; align: `start` \| `center` \| `end`; } | - | 5.13.0 |
 | items | Configure tab content | [TabItemType](#tabitemtype) | [] | 4.23.0 |
 | moreIcon | The custom icon of ellipsis | ReactNode | `<EllipsisOutlined />` | 4.14.0 |
-| removeIcon | The custom icon of remove | ReactNode | `<CloseOutlined />` | 5.15.0 |
+| removeIcon | The custom icon of remove, Only works while `type="editable-card"` | ReactNode | `<CloseOutlined />` | 5.15.0 |
```
Comment (MadCcc) [inline components/tabs/index.en-US.md:59]:
```suggestion
| removeIcon | The custom icon of remove, only works with `type="editable-card"` | ReactNode | `<CloseOutlined />` | 5.15.0 |
```

**Review by MadCcc — APPROVED** [review]

**Pull request merged**